deltachat-ios/Controller/WelcomeViewController.swift

```diff
@@ -316,7 +322,9 @@ extension WelcomeViewController: QrCodeReaderDelegate {
     }
 
     private func confirmAccountCreationAlert(accountDomain domain: String, qrCode: String) {
-        let title = String.localizedStringWithFormat(String.localized("qraccount_ask_create_and_login"), domain)
+        let title = String.localizedStringWithFormat(
+            String.localized(dcAccounts.getAll().count > 1 ? "qraccount_ask_create_and_login_another" : "qraccount_ask_create_and_login"),
```

Comment:
@r10s I've found in the resources that there's an extra string for the case there's already an account configured.We didn't use that string before.